test/unit/reducer/tests/test-reducer-constructors.hpp

```diff
@@ -175,6 +204,9 @@ TYPED_TEST_P(ReducerInitConstructorUnitTest, InitReducerConstructor)
   testInitReducerConstructor< ReduceType, NumericType, ResourceType, ForOneType >();
 }
 
+#if defined(RAJA_ENABLE_HIP)
```

Comment:
Should this be #if !defined?

Reply:
Found a better fix for this. Turns out I forgot to activate the HIP basic test, which should be fixed in the next commit.

Reply:
Ah. Good catch. Does this work for CUDA?

Reply:
Yes, this works for CUDA as well.